pjmedia/include/pjmedia/transport_ice.h

```diff
@@ -152,7 +152,17 @@ enum pjmedia_transport_ice_options
      * are different after several packets are received.
      * Specifying this option will disable this feature.
      */
-    PJMEDIA_ICE_NO_SRC_ADDR_CHECKING = 1
+    PJMEDIA_ICE_NO_SRC_ADDR_CHECKING = 1,
+
+    /**
+     * The standard (rfc5245) specify that ice-mismatch attribute is used
+     * due to a mismatch of candidates with the default destination for media
+     * signaled in the SDP. However rfc8445 specify that ALGs, can alter
+     * signaling information in ways that break ICE and triggered ice mismatch.
+     * (e.g., by rewriting IP addresses in SDP). So disabling ice mismatch
+     * is required on some scenario.
```

Comment:
The rfc5245 seems to have identified the ALG issues as well:
> Unfortunately, many ALGs are known to work poorly in these corner
>    cases.  ICE does not try to work around broken ALGs, as this is
>    outside the scope of its functionality.  ICE can help diagnose these
>    conditions, which often show up as a mismatch between the set of
>    candidates and the m and c lines and rtcp attributes.  The ice-
>    mismatch attribute is used for this purpose.

And rfc8445 does not seems to cancel ice-mismatch rule.

So I think the docs needs to be updated.